batchflow/research/research.py

```diff
@@ -19,14 +22,32 @@
 from .domain import Domain, Option, ConfigAlias
 from .job import Job
 from .logger import BaseLogger, FileLogger, PrintLogger, TelegramLogger
-from .utils import get_metrics
-from .executable import Executable
+from .utils import get_metrics, count_startswith
+from .executable import Executable, Namespace
 from .named_expr import RP
 
 class Research:
     """ Class Research for multiple parallel experiments with pipelines. """
-    def __init__(self):
+    def __init__(self, domain=None, n_configs=None, n_reps=1, repeat_each=100):
```

Comment:
As this signature is actually visible and tabbable in notebooks, let's add more documentation here (at least for other parameters)